src/main/java/com/snapscore/pipeline/pulling/http/VertxClientCallbackImpl.java

```diff
@@ -53,6 +65,7 @@ public void onResponse(HttpClientResponse response) {
             } else {
                 handleUnsuccessfulResponse(response.statusCode());
             }
+            headersObservers.forEach(ho -> ho.observeHeaders(feedRequest.getFeedName(), response.headers()));
```

Comment:
@petrsnapscore mohli bychom tohle obalit do samostatneho try-catch blocku? Je to kvuli tomu, ze pokud to failne, tak kod nad tim uz mohl vyemitovat successful response do fluxu, ktery tohle ridi. Bez vnoreneho try-catch blocku by se pak jeste vyemotivala Exception z venejsiho try-catch a to by byl problem

Reply:
Chápu, obaleno